pubspec.lock

```diff
@@ -41,6 +41,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "1.17.2"
+  crypto:
```

Comment:
What does this dependency do in the current PR?

Reply:
i am not too sure what dependency is that, but how i work is, i add the required dependencies through pubspec.yaml
so when you add a dependency flutter updates the pubspec.lock automatically so i am not sure which dependency came through with crypto